nntrainer/graph/network_graph.cpp

```diff
@@ -748,7 +748,8 @@ NetworkGraph::finalizeContext(const std::shared_ptr<LayerNode> &lnode,
 
   lnode->configureRunContext(
     // TODO: update weights spec for trainable based on layer trainable prop
```

Comment:
you can remove this TODO :)

Reply:
Thought about this, I am going to do this todo later. We will be only passing weight spec, not the additional parameter like trainable :)

Reply:
alright.